lib/src/screens/session/disclosure/bloc/disclosure_permission_bloc.dart

```diff
@@ -320,8 +324,17 @@ class DisclosurePermissionBloc extends Bloc<DisclosurePermissionBlocEvent, Discl
         return DisclosurePermissionFinished();
       }
       return state;
-    } else if (state is DisclosurePermissionIssueWizard) {
-      final refreshedState = _refreshIssueWizard(state, session);
+    } else if (state is DisclosurePermissionIssueWizard ||
+        (state is DisclosurePermissionCredentialInformation && state.parentState is DisclosurePermissionIssueWizard)) {
+      final DisclosurePermissionIssueWizard issueWizardState;
+      if (state is DisclosurePermissionIssueWizard) {
+        issueWizardState = state;
+      } else {
+        issueWizardState =
+            (state as DisclosurePermissionCredentialInformation).parentState as DisclosurePermissionIssueWizard;
+      }
```

Comment:
Didn't you forget the `DisclosurePermissionObtainCredentials` state? I think there are also causes that we have to return from `DisclosurePermissionCredentialInformation` to `DisclosurePermissionObtainCredentials`.

Reply:
You are right! I added it to the flow and test.